Core/Graphics/Source/Graphics.cpp

```diff
@@ -277,6 +286,11 @@ namespace Babylon
         m_impl->SetNativeWindow(windowPtr);
     }
 
+    void Graphics::UpdateWindowType(void* windowType)
```

Comment:
Would you ever update the window type after creating the graphics object? Seems like this can be part of UpdateWindow and CreateGraphics (possibly as overloads)?

Reply:
no, this seems like a good suggestion

Reply:
this should now be updated